.gitignore

```diff
@@ -19,6 +19,7 @@ openlibrary.egg-info/
 pip-log.txt
 supervisord.pid
 vendor/apache-solr-1.4.0/
+vendor/infogami/
```

Comment:
I don't think this will do what we want 😬 If the directory has already been committed (which this has been), this doesn't prevent the file from showing up as modified.